### PR TITLE
Explo Data Visualization in Docs

### PR DESCRIPTION
**Pull request by brianexplo**

### CODE REVIEW

**Comment by DanRoscigno** [comment]
I am good with this if you are @mshustov 

Thanks for this @brianexplo !

File: docs/en/integrations/index.mdx [hunk 1]
```diff
@@ -71,6 +71,7 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
 |dbt|<DbtSVG style={{width: '3rem'}} />|Data ingestion|Use dbt (data build tool) to transform data in ClickHouse by simply writing select statements. dbt puts the T in ELT.|[Documentation](/docs/en/integrations/data-ingestion/etl-tools/dbt/index.md)|
 |DeltaLake|<img src={require('./images/logos/deltalake.png').default} class="image" alt="Delta Lake logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data integration| provides a read-only integration with existing [Delta Lake](https://github.com/delta-io/delta) tables in Amazon S3.|[Documentation](/docs/en/engines/table-engines/integrations/deltalake)|
 |EmbeddedRocksDB|<RocksdbSVG style={{width: '3rem', 'height': '3rem'}} />|Data integration|Allows integrating ClickHouse with [rocksdb](http://rocksdb.org/).|[Documentation](/docs/en/engines/table-engines/integrations/embedded-rocksdb)|
+|Explo|<img src={require('./images/logos/explo.png').default} class="image" alt="Explo logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data visualization|Explo is a customer-facing analytics tool for any platform|[Documentation](/docs/en/integrations/explo)|
```
Comment (mshustov) [inline docs/en/integrations/index.mdx:74]: Could you move it to the `Partner integrations` table?

File: docusaurus.config.js [hunk 2]
```diff
@@ -407,6 +407,10 @@ const config = {
 						from: '/en/connect-a-ui/metabase-and-clickhouse',
 						to: '/en/integrations/metabase',
 					},
+					{
+						from: '/en/connect-a-ui/explo-and-clickhouse',
```
Comment (mshustov) [inline docusaurus.config.js:411]: I don't think we need this redirect for new pages

File: docs/en/integrations/data-visualization/explo-and-clickhouse.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+<img src={require('./images/explo_04.png').default} class="image" alt="Clickhouse" />
+
+
+6. Enter your **Clickhouse Credentials**. 
+
+
+<img src={require('./images/explo_05.png').default} class="image" alt="Credentials" />
+
+
+7. Configure **Security**, and you are done!
```
Comment (mshustov) [inline docs/en/integrations/data-visualization/explo-and-clickhouse.md:60]: Could you provide a link to a page with all the Explo egress IPs? These IP must be whitelisted in ClickHouse Cloud

Comment (brianexplo) [reply docs/en/integrations/data-visualization/explo-and-clickhouse.md]: Done!

File: docusaurus.config.js (hunk 2 above)
Comment (brianexplo) [reply docusaurus.config.js]: Done!

**Pull request merged**